### DISCUSSION TITLE
Stradella Bass System as an Additional Pianokey View

### DISCUSSION THREAD

**Discussion by motrojam** [SynthstromAudible/DelugeFirmware#1942] [Ideas]
I'm loving the deluge and that it's open source now. I have no idea about programming so just throwing this out there in the hope it reaches someone who can actually implement this idea... In addition to the keyboard layout it would be amazing if there was a stradella bass system view. I've loved messing around with accordions bass buttons for the complex chord voicings that can be created with the stradella system. I've also recently come across chord organs which look cool but have the downside of a fan making noise. It would absolutely rock my world if stradella system could be implemented into a community firmware down the track. 

Looking at a 120 bass layout on an accordion this wouldn't quite fit on the deluge but.. it probably doesn't need to because there's always the option of side scrolling. From what I understand, accordion players generally don't jump from one end of the bass system to the other.. the 120 bass allows for common chord progressions to be played close together on the buttons. Also given that a stradella system only has 6 rows (2bass + 4chords) perhaps there is space on the grid for two rows of melodic input - similar to the button system on a diatonic accordion perhaps? But with the option of the setting the scale (i.e. so it's not limited to diatonic)? I suppose I'm imagining the stradella system as a possible live performance mode so this could be getting a bit busy and it might actually be more interesting to use any remaining button rows to innovate on the stradella system with extra chord formations although I'm sure real accordionists know the tricks for creating any chord they want from the major, minor, dominant and diminished available. I don't have an exact idea of how this should look but I would love to see what someone comes up with.

**Comment by motrojam** [comment]
PS - Just realising this post reads as I want the deluge to emulate an accordion. I mean that would be cool but there's lots going on with accordions in terms of expressiveness of the bellows that I realise goes well beyond just having a stradella bass system. I'm mainly interested in sharing this idea as a way of transferring skills/ knowledge gained from acoustic instruments to the deluge. It's pretty obvious that a synth or keyboard manufacture is never going to revisit the chord organ style instruments of the 50s and 60s. Maybe the community that's building around the deluge is the opening for this curiosity/ crossover between accordions/ harmoniums, hammond s6 to be revisited.

**Comment by nikodemus** [comment]
Had to look it up 😅https://en.m.wikipedia.org/wiki/Stradella_bass_system
